### ISSUE TITLE
[BUG] Items duplicate themselves then shoot out of my inventory 

### ISSUE THREAD

**Issue by AnthonyEZ917** [tsunamayo/Starship-EVO#6077]
### Short Description of the issue:

I’m just playing normally, but sometimes when I put items in storage, they start duplicating and shoot out of my inventory, flooding my base. After a game restart, they do disappear.

### Steps to reproduce the issue:

I am unsure it happens at random and I have tried to recreate it a few time now.

### Environment
- Starship EVO version: early-access 24w42b
- Graphics Card: NVIDIA GeForce RTX 4070 Ti

### Additional information:

[Survival2.zip](https://github.com/user-attachments/files/17564236/Survival2.zip)

https://github.com/user-attachments/assets/3f92660f-7f62-4764-97ec-24dafe80b63e

**Comment by BigBadKangaroo** [comment]
I remember this one :'D #5562

**Comment by tsunamayo** [comment]
@AnthonyEZ917 Thanks, yes it is hard to reproduce. Do you recall doing anything at the same time? Any assembler, collector or furnace working? I need a lead to understand how to reproduce. Thanks

**Comment by AnthonyEZ917** [comment]
I think it started doing it when when I turned on assemblers work together. as I didn't have the problem for a while before i set that up.

**Comment by AnthonyEZ917** [comment]
Yeah, it happened again, but only after I turned on the assemblers share queue. That seems to be causing it.

**Comment by AnthonyEZ917** [comment]
Oh yeah, another thing sometimes it fills an assembler with grass for some reason.
![image](https://github.com/user-attachments/assets/eed875bc-3ea0-48b1-ae10-0d5526237de6)
